app/lib/features/chat_ng/dialogs/message_actions.dart

```diff
@@ -22,29 +28,36 @@ void reactionSelectionAction({
     barrierColor: Colors.transparent,
     transitionDuration: const Duration(milliseconds: 200),
     pageBuilder: (context, animation, secondaryAnimation) {
-      return Stack(
+      return Column(
+        mainAxisAlignment: MainAxisAlignment.center,
+        crossAxisAlignment: CrossAxisAlignment.center,
         children: [
-          _ReactionOverlay(
+          _BlurOverlay(
             animation: animation,
-            child: const SizedBox.expand(),
+            child: const SizedBox.shrink(),
           ),
-          Positioned(
-            left: position.dx,
-            top: position.dy,
-            width: messageSize.width,
-            child: messageWidget,
+          // Reaction Row
+          _AnimatedActionsContainer(
+            animation: animation,
+            tagId: messageId,
+            child: ReactionSelector(
+              isMe: isMe,
+              messageId: '$messageId-reactions',
+              roomId: roomId,
+            ),
           ),
-          Positioned(
-            left: position.dx,
-            top: position.dy - 60,
-            child: _AnimatedReactionSelector(
```

Comment:
This is evolved to single  re-usable ```AnimatedActionsContainer``` because we need similar effect for message actions :)